### PR TITLE
sys-auth/fprintd: bumping to version 1.90.9

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.17, Repoman-3.0.2
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: sys-auth/fprintd/fprintd-1.90.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+EAPI=7
+
+inherit meson pam systemd
+
+DESCRIPTION="D-Bus service to access fingerprint readers"
+HOMEPAGE="https://gitlab.freedesktop.org/libfprint/fprintd"
+SRC_URI="https://gitlab.freedesktop.org/libfprint/${PN}/-/archive/v${PV}/${PN}-v${PV}.tar.bz2"
+
+LICENSE="GPL-2"
```
Comment (juippis) [inline sys-auth/fprintd/fprintd-1.90.9.ebuild:12]: Seems to be `GPL-2+`. At least the 1.90.8 is also gpl-2+, could you check 0.8 versions too, and update all relevant ebuilds with updated LICENSE field in a **separate** commit.

Comment (scardracs) [reply sys-auth/fprintd/fprintd-1.90.9.ebuild:12]: @juippis I have requested stabilization for 1.90.8. Can I change the license without break the stabilization request?

Comment (juippis) [reply sys-auth/fprintd/fprintd-1.90.9.ebuild:12]: Yes, it doesn't affect runtime or list of installed files.

Comment (scardracs) [reply sys-auth/fprintd/fprintd-1.90.9.ebuild:12]: Ok, I change them now

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**